.travis.yml

```diff
@@ -3,5 +3,9 @@ rvm:
   - 2.3
   - 2.6
 env:
-  - TESTING_INTERPRETER=true
-  - TESTING_INTERPRETER=false
+  -
+    GRAPHQL_VERSION: 1.9.19
+    TESTING_LEGACY_MUTATION: true
```

Comment:
Just a naming nitpick but I'd probably generalize this a bit: `TESTING_LEGACY_DEFINITION_LAYER`

Reply:
Amended.